spinalcordtoolbox/scripts/sct_dmri_moco.py

```diff
@@ -161,6 +167,12 @@ def get_parser():
         metavar=Metavar.str,
         help="If provided, this string will be mentioned in the QC report as the subject the process was run on."
     )
+    optional.add_argument(
+        '-mocodl',
```

Comment:
I am not sure about the name `-mocodl`, just because `moco` is already in the name of the script itself (e.g. `sct_dmri_moco -mocodl` is a little strange).

Maybe just `-dl` would be simpler?